frontend/src/metabase/palette/hooks/useCommandPalette.tsx

```diff
@@ -124,6 +124,10 @@ export const useCommandPalette = ({
     showDocsAction,
   ]);
 
+  const metabotActions =
```

Comment:
This can live here, but you could also move it out into it's own hook if that makes things simpler. I think at one point, we were encouraged to keep all Metabot stuff in one place so it was easy to remove if needed.

Reply:
Hm, how would I add palette the Metabot palette action from the "outside" - from my own hook in that case?